Add Aeronautical Charts Ordering, Pricing, Invoicing & Notifications

Doro256 opened the pull request:
Created AeronauticalCharts component and Incorporated it for pricing & order reference

Doro256 committed:
remove Checklists

Doro256 committed:
remove Checklists

Doro256 committed:
Fix visual truncation of product descriptions

Doro256 committed:
Fix visual truncation of product descriptions

Doro256 committed:
add AeronauticalCharts component with pricing & order table

Doro256 committed:
add AeronauticalCharts component with pricing & order table

Doro256 committed:
add AeronauticalCharts component with pricing & order table

Doro256 committed:
add AeronauticalCharts component with pricing & order table

Doro256 committed:
Incorporate AeronauticalCharts component for pricing & order reference

Doro256 committed:
Incorporate AeronauticalCharts component for pricing & order reference

Doro256 committed:
Incorporate AeronauticalCharts component for pricing & order reference

Doro256 committed:
Incorporate AeronauticalCharts component for pricing & order reference

Doro256 committed:
Incorporate AeronauticalCharts component for pricing & order reference

Doro256 committed:
Incorporate AeronauticalCharts component for pricing & order reference

Doro256 committed:
Include instrument approach, departure, and arrival charts

Add parking/docking, enroute, and index charts

Implement proper categorization and pricing structure

Doro256 committed:
Include instrument approach, departure, and arrival charts

Add parking/docking, enroute, and index charts

Implement proper categorization and pricing structure

Doro256 committed:
Include instrument approach, departure, and arrival charts

Add parking/docking, enroute, and index charts

Implement proper categorization and pricing structure

Doro256 committed:
Include instrument approach, departure, and arrival charts

Add parking/docking, enroute, and index charts

Implement proper categorization and pricing structure

Doro256 committed:
Include instrument approach, departure, and arrival charts

Add parking/docking, enroute, and index charts

Implement proper categorization and pricing structure

Doro256 committed:
Include instrument approach, departure, and arrival charts

Add parking/docking, enroute, and index charts

Implement proper categorization and pricing structure

Doro256 committed:
Enhance dropdown visibility with high-contrast styling

Doro256 committed:
Enhance dropdown visibility with high-contrast styling

Doro256 committed:
Modified styling

Doro256 committed:
Modified styling

Doro256 committed:
Modified styling

Doro256 committed:
Modified styling

Doro256 committed:
Modified styling

Doro256 committed:
Modified styling

Doro256 committed:
Created ChartOrderForm component

Doro256 committed:
Created ChartOrderForm component

Doro256 committed:
Created ChartOrderForm component

Doro256 committed:
Created new component for manageing chart orders by admin

Doro256 committed:
Created new component for manageing chart orders by admin

Doro256 committed:
Created new component for manageing chart orders by admin

Doro256 committed:
created component to manage chart orders

Doro256 committed:
created component to manage chart orders

Doro256 committed:
created component to manage chart orders

Doro256 committed:
Added route paths for charts management

Doro256 committed:
Added route paths for charts management

Doro256 committed:
Added route paths for charts management

Doro256 committed:
Added route paths for charts management

Doro256 committed:
Added route paths for charts management

Doro256 committed:
Added route paths for charts management

Doro256 committed:
added chart management icons on the Sidebar
updated  apiSlice to fetch charts data from the backend

Doro256 committed:
added chart management icons on the Sidebar
updated  apiSlice to fetch charts data from the backend

Doro256 committed:
added chart management icons on the Sidebar
updated  apiSlice to fetch charts data from the backend

JonahKayizzi committed:
Merge pull request #9 from JonahKayizzi/remove-checklists

Remove free resources (NOTAM, PIB, Checklists, NOTAM lists) from subscription flow

JonahKayizzi committed:
Merge branch 'grey-text' of https://github.com/JonahKayizzi/subscription-upgrade-front into grey-text

JonahKayizzi committed:
Merge pull request #10 from JonahKayizzi/grey-text

Fix incomplete grey description for Aeronautical Information Products

JonahKayizzi committed:
Merge branch 'charts' of https://github.com/JonahKayizzi/subscription-upgrade-front into charts

JonahKayizzi commented:
The cards on this page have rounded edges. Please maintain

<img width="1077" height="868" alt="image" src="https://github.com/user-attachments/assets/3c6264be-8717-4bf2-9ebf-9143edb5325d" />

JonahKayizzi commented:
Place the charts subscription card below the aip prices card

JonahKayizzi commented:
The charts card is a color salad. Let us maintain it within the same color theme as the producrs card or the subscription form

JonahKayizzi committed:
fix: usedispatch error

JonahKayizzi commented:
Your changes have modified the dashboard design

Your dashboard
<img width="1908" height="993" alt="image" src="https://github.com/user-attachments/assets/6070f305-1b64-4a49-9b0a-3014a7cf1bc5" />

The old dashboard

<img width="1902" height="983" alt="image" src="https://github.com/user-attachments/assets/e94919ef-a40e-420d-95ac-92b4cc41b9d5" />

JonahKayizzi commented:
I am not getting the purpose of this route 

<img width="1918" height="975" alt="image" src="https://github.com/user-attachments/assets/496d4196-78a7-4b1b-8af9-0d0b81dc67df" />

JonahKayizzi committed:
Merge branch 'subscription-type-preselected' of https://github.com/JonahKayizzi/subscription-upgrade-front into subscription-type-preselected

JonahKayizzi committed:
Merge pull request #15 from JonahKayizzi/subscription-type-preselected

Font Color and Bg Color for drop downs

Doro256 committed:
Create component for exporting annual subscription report in excel

Doro256 committed:
Create button for handling Excel export

Doro256 committed:
Add API endpoint for the annual subscription report

Doro256 committed:
add the Excel export button to the Dashboard component

Doro256 committed:
Add xlsx dependency to package.json

Doro256 committed:
Update lock file

Doro256 committed:
update all the pricing in the App.js file to include 18% VAT

Doro256 committed:
add a VAT disclaimer to the subscription form

Doro256 committed:
Update AdminSubscriptionRequestModal to accept and use pre-selected subscription type

Doro256 committed:
Update AdminSubscriptionRequestModal to accept and use pre-selected subscription type

Doro256 committed:
modify the Dashboard component to pass the subscription type

Doro256 committed:
modify the Dashboard component to pass the subscription type

Doro256 committed:
modify the SubscribersTable to find the expiring subscription and pass its type

Doro256 committed:
modify the SubscribersTable to find the expiring subscription and pass its type

Doro256 committed:
move the notifications card above the statistics section

Doro256 committed:
move the notifications card above the statistics section

Doro256 committed:
Replace static notification text with dynamic data from API

Doro256 committed:
Replace static notification text with dynamic data from API

Doro256 committed:
Updated PR with fixes

Doro256 committed:
create a utility function to generate mailing labels

Doro256 committed:
create a utility function to generate mailing labels

Doro256 committed:
Update function to integrate printing labels

Doro256 committed:
Update function to integrate printing labels

Doro256 committed:
add endpoint for fetching paper subscriptions for mailing labels

Doro256 committed:
add endpoint for fetching paper subscriptions for mailing labels

JonahKayizzi committed:
Merge pull request #25 from JonahKayizzi/vat-disclaimer

Vat disclaimer

JonahKayizzi committed:
Merge branch 'renewal-request' of https://github.com/JonahKayizzi/subscription-upgrade-front into renewal-request

JonahKayizzi committed:
Merge pull request #26 from JonahKayizzi/renewal-request

Renewal request with pre-selected subscription type

JonahKayizzi committed:
Merge branch 'subscriber-notifications' of https://github.com/JonahKayizzi/subscription-upgrade-front into subscriber-notifications

JonahKayizzi committed:
Merge pull request #27 from JonahKayizzi/subscriber-notifications

Recent Subscriber notifications

JonahKayizzi committed:
update this branch with dev

JonahKayizzi committed:
update this branch with dev

JonahKayizzi committed:
Merge branch 'annual-subscription-report' of https://github.com/JonahKayizzi/subscription-upgrade-front into annual-subscription-report

JonahKayizzi committed:
Merge pull request #24 from JonahKayizzi/annual-subscription-report

feat: Print Excel sheet of the annual subscription report

JonahKayizzi committed:
Merge branch 'mailing-labels' of https://github.com/JonahKayizzi/subscription-upgrade-front into mailing-labels

JonahKayizzi committed:
merge dev to current branch

JonahKayizzi committed:
Merge pull request #28 from JonahKayizzi/mailing-labels

Print Mailing labels

JonahKayizzi committed:
Merge branch 'charts' of https://github.com/JonahKayizzi/subscription-upgrade-front into charts

JonahKayizzi committed:
merge dev to current branch

JonahKayizzi commented:
Endpoint for chart orders is missing in the backend
Also provide database modfications for chart orders

JonahKayizzi committed:
correct dashboard layout

JonahKayizzi reviewed (commented):
@Doro256 , I have handled the conflicts on this branch
You can proceed with the work on this branch

Doro256 committed:
dd - charts browsing and orders UI

- Include chart catalog browsing, ordering updates, subscriber access, and admin chart management UI updates.

Doro256 commented:
Closing because we will merge this into dev from the correct PR.
